### PR TITLE
Added a few papers from UPenn for this year

### PR DESCRIPTION
**Pull request by alexzzhu**
I've put our EV-FlowNet paper under optical flow and pattern recognition. Please feel free to move it around as you see fit. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -373,6 +375,9 @@ IEEE Int. Symp. Circuits and Systems (ISCAS), Baltimore, MD, 2017.
 - <a name="Haessig17arxiv"></a>Haessig, G., Cassidy, A. Alvarez, R., Benosman, R., Orchard, G.,  
 *[Spiking Optical Flow for Event-based Sensors Using IBM's TrueNorth Neurosynaptic System](https://arxiv.org/pdf/1710.09820.pdf)*,  
 arXiv:1710.09820, 2017.
+- <a name="Zhu2018rss"></a>Zhu, A., Yuan, L., Chaney, K., Daniilidis, K.,
```
Comment (guillermogb) [inline README.md]: Zhu18rss

File: README.md [hunk 2]
```diff
@@ -478,8 +483,8 @@ IEEE Robotics and Automation Letters (RA-L), 2:2(632-639), 2017.
 - [Mueggler et. al. IJRR 2017](#Mueggler17ijrr).  
 *The Event-Camera Dataset and Simulator: Event-based Data for Pose Estimation, Visual Odometry, and SLAM.*
 - <a name="Zhu17cvpr"></a>Zhu, A., Atanasov, N., Daniilidis, K.,  
-*Event-based Visual Inertial Odometry*,  
-IEEE Conf. Computer Vision and Pattern Recognition (CVPR) 2017. [PDF](http://openaccess.thecvf.com/content_cvpr_2017/papers/Zhu_Event-Based_Visual_Inertial_CVPR_2017_paper.pdf), [Supplementary material](http://openaccess.thecvf.com/content_cvpr_2017/supplemental/Zhu_Event-Based_Visual_Inertial_2017_CVPR_supplemental.zip)
+*[Event-based Visual Inertial Odometry](http://openaccess.thecvf.com/content_cvpr_2017/papers/Zhu_Event-Based_Visual_Inertial_CVPR_2017_paper.pdf)*,
```
Comment (guillermogb) [inline README.md]: The link on the title of the paper points to the official IEEE link, which is given by the DOI.
The PDF, if available, comes later, as you can see.

File: README.md [hunk 3]
```diff
@@ -617,6 +625,9 @@ Int. J. Robotics Research, 36:2, pp. 142-149, 2017. [PDF](https://arxiv.org/pdf/
 - <a name="Binas17icml"></a>Binas, J., Neil, D., Liu, S.-C., Delbruck, T.,  
 *[DDD17: End-To-End DAVIS Driving Dataset](https://www.openreview.net/pdf?id=HkehpKVG-),*  
 Int. Conf. Machine Learning, Sydney, Australia, PMLR 70, 2017. [Dataset](http://sensors.ini.uzh.ch/databases.html)
+- <a name="Zhu18mvsec"></a>Zhu, A., Thakur, D., Ozaslan, T., Pfrommer, B., Kumar, V., Daniilidis, K.,
```
Comment (guillermogb) [inline README.md]: Zhu18ral. 
And the link is to the official page pointed by the DOI.

File: README.md [hunk 4]
```diff
@@ -194,8 +194,8 @@ IEEE/RSJ Int. Conf. Intelligent Robots and Systems (IROS), Daejeon, South Korea,
 *[A Motion-Based Feature for Event-Based Pattern Recognition](https://doi.org/10.3389/fnins.2016.00594)*.  
 Front. Neurosci. (2017), 10:594.
 - <a name="Zhu17icra"></a>Zhu, A., Atanasov, N., Daniilidis, K.,  
-*[Event-based Feature Tracking with Probabilistic Data Associations](https://fling.seas.upenn.edu/~alexzhu/dynamic/event-based-feature-tracking-with-probabilistic-data-association/)*,  
-IEEE Int. Conf. Robotics and Automation (ICRA), Singapore, 2017. [YouTube](https://youtu.be/m93XCqAS6Fc)
+*[Event-based Feature Tracking with Probabilistic Data Association](https://fling.seas.upenn.edu/~alexzhu/dynamic/event-based-feature-tracking-with-probabilistic-data-association/)*,
```
Comment (guillermogb) [inline README.md]: The link on the title of the paper points to the official IEEE link, which is given by the DOI.
The PDF, if available, should go later, before the YouTube link.

File: README.md [hunk 5]
```diff
@@ -373,6 +375,9 @@ IEEE Int. Symp. Circuits and Systems (ISCAS), Baltimore, MD, 2017.
 - <a name="Haessig17arxiv"></a>Haessig, G., Cassidy, A. Alvarez, R., Benosman, R., Orchard, G.,  
 *[Spiking Optical Flow for Event-based Sensors Using IBM's TrueNorth Neurosynaptic System](https://arxiv.org/pdf/1710.09820.pdf)*,  
 arXiv:1710.09820, 2017.
+- <a name="Zhu2018rss"></a>Zhu, A., Yuan, L., Chaney, K., Daniilidis, K.,  
+*[EV-FlowNet: Self-Supervised Optical Flow Estimation for Event-based Cameras](https://arxiv.org/abs/1802.06898)*,
```
Comment (guillermogb) [inline README.md]: The link on the title of the paper should be the one on the RSS proceedings, once you know the DOI.
The PDF, if available, should go later, before the YouTube link.

File: README.md [hunk 6]
```diff
@@ -559,6 +564,9 @@ IEEE Int. Symp. Circuits and Systems (ISCAS), 2017.
 *[HATS: Histograms of Averaged Time Surfaces for Robust Event-based Object Classification](https://arxiv.org/abs/1803.07913)*,  
 IEEE Conf. Computer Vision and Pattern Recognition (CVPR) 2018. 
     - [N-CARS Dataset](#ncars_dataset): A large real-world event-based dataset for car classification.
+- <a name="Zhu2018rss"></a>Zhu, A., Yuan, L., Chaney, K., Daniilidis, K.,
```
Comment (guillermogb) [inline README.md]: This entry is duplicated. Please link to the primary entry in the optical flow section. See the example from Bardow's entry, which is also in more than one category.

**Comment by guillermogb** [comment]
Thanks for contributing to the list!

File: README.md (hunk 4 above)
Comment (alexzzhu) [reply README.md]: Updated with doi

File: README.md [hunk 7]
```diff
@@ -319,7 +319,9 @@ IEEE Trans. Neural Netw. Learn. Syst., 2017.
 - <a name="Xie17fnins"></a>Xie, Z., Chen, S., Orchard, G.  
 *[Event-Based Stereo Depth Estimation Using Belief Propagation](https://doi.org/10.3389/fnins.2017.00535)*,  
 Front. Neurosci. (2017), 11:535.  [YouTube](https://youtu.be/ngJpY1lcbdw)
-
+- <a name="Zhu2018realtime"></a>Zhu, A., Chen, Y., Daniilidis, K.,
```
[the remark this answers is not in the document]
Comment (alexzzhu) [reply README.md]: Updated

File: README.md (hunk 5 above)
Comment (alexzzhu) [reply README.md]: I've removed the link for now and added a PDF link. Will update with doi once available.

File: README.md (hunk 6 above)
Comment (alexzzhu) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (alexzzhu) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (alexzzhu) [reply README.md]: Updated

File: README.md [hunk 8]
```diff
@@ -373,6 +375,9 @@ IEEE Int. Symp. Circuits and Systems (ISCAS), Baltimore, MD, 2017.
 - <a name="Haessig17arxiv"></a>Haessig, G., Cassidy, A. Alvarez, R., Benosman, R., Orchard, G.,  
 *[Spiking Optical Flow for Event-based Sensors Using IBM's TrueNorth Neurosynaptic System](https://arxiv.org/pdf/1710.09820.pdf)*,  
 arXiv:1710.09820, 2017.
+- <a name="Zhu18rss"></a>Zhu, A., Yuan, L., Chaney, K., Daniilidis, K.,  
+*EV-FlowNet: Self-Supervised Optical Flow Estimation for Event-based Cameras*,  
+Robotics, Science and Systems 2018. [PDF](https://arxiv.org/abs/1802.06898), [YouTube](https://youtu.be/eMHZBSoq0sE)
```
Comment (guillermogb) [inline README.md]: Thanks. Can you write the same format as our RSS'15 paper? Thanks. In your case:
Robotics: Science and Systems XIV (RSS), Pittsburgh (PA), USA, 2018.

File: README.md [hunk 9]
```diff
@@ -559,6 +564,8 @@ IEEE Int. Symp. Circuits and Systems (ISCAS), 2017.
 *[HATS: Histograms of Averaged Time Surfaces for Robust Event-based Object Classification](https://arxiv.org/abs/1803.07913)*,  
 IEEE Conf. Computer Vision and Pattern Recognition (CVPR) 2018. 
     - [N-CARS Dataset](#ncars_dataset): A large real-world event-based dataset for car classification.
+- [Zhu et. al. RSS 2018](#Zhu18rss),
+*EV-FlowNet: Self-Supervised Optical Flow Estimation for Event-based Cameras.*
```
Comment (guillermogb) [inline README.md]: Line break here, please. Authors and paper title in different lines.

File: README.md [hunk 10]
```diff
@@ -470,16 +475,16 @@ IEEE Robotics and Automation Letters (RA-L), 2:2(632-639), 2017.
 - [Mueggler et. al. IJRR 2017](#Mueggler17ijrr).  
 *The Event-Camera Dataset and Simulator: Event-based Data for Pose Estimation, Visual Odometry, and SLAM.*
 
-<a name="visual-inertial"></a>
+<a name="f"></a>
```
Comment (guillermogb) [inline README.md]: Please revert this change.  It breaks the link in the table of contents.

File: README.md [hunk 11]
```diff
@@ -617,6 +624,9 @@ Int. J. Robotics Research, 36:2, pp. 142-149, 2017. [PDF](https://arxiv.org/pdf/
 - <a name="Binas17icml"></a>Binas, J., Neil, D., Liu, S.-C., Delbruck, T.,  
 *[DDD17: End-To-End DAVIS Driving Dataset](https://www.openreview.net/pdf?id=HkehpKVG-),*  
 Int. Conf. Machine Learning, Sydney, Australia, PMLR 70, 2017. [Dataset](http://sensors.ini.uzh.ch/databases.html)
+- <a name="Zhu18mvsec"></a>Zhu, A., Thakur, D., Ozaslan, T., Pfrommer, B., Kumar, V., Daniilidis, K.,  
+*[The Multi Vehicle Stereo Event Camera Dataset: An Event Camera Dataset for 3D Perception](https://doi.org/10.1109/LRA.2018.2800793),*  
+IEEE Robotics and Automation Letters 3.3 (2018): 2032-2039. [PDF](https://arxiv.org/abs/1801.10202), [PDF RA-L](http://doi.org/10.1109/LRA.2018.2800793), [Dataset](https://daniilidis-group.github.io/mvsec/), [YouTube](https://youtu.be/9FaUvvzaHW8).
```
Comment (guillermogb) [inline README.md]: I do not think that you need the second PDF (PDF RA-L); the link is already in the title of the paper.

**Comment by guillermogb** [comment]
Thanks! Minor comments to be addressed.

File: README.md (hunk 1 above)
Comment (alexzzhu) [reply README.md]: Updated.

File: README.md (hunk 11 above)
Comment (alexzzhu) [reply README.md]: Removed.

File: README.md (hunk 10 above)
Comment (alexzzhu) [reply README.md]: Oops, this was unintended, I've reverted the change.

File: README.md (hunk 9 above)
Comment (alexzzhu) [reply README.md]: Updated.

File: README.md (hunk 8 above)
Comment (alexzzhu) [reply README.md]: Updated.

**Comment by guillermogb** [comment]
Merged. Thanks!
